infrastructure/kube/keep-test/maintainer/bitcoin-difficulty-statefulset.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
```

Comment:
Could we use kustomization for setup? Please see https://github.com/keep-network/keep-core/blob/main/infrastructure/kube/keep-test/tbtc-v2-maintainer/kustomization.yaml for reference.

Reply:
Done in 2b95e135407482fb1d9c53aef342dedc01d99701.